scripts/seed_users.js

```diff
@@ -0,0 +1,414 @@
+// Script to import users from csv
+
+// Read the name of the file
+var filename = process.argv.slice(2)[0];
+if (filename) {
+    console.log("File: " + filename);
+} else {
+    console.log('Filename not specified');
+    process.exit(-1);
+}
+
+// Import csv-parser module
+var csv = require('csv-parser');
+
+// Import fs module
+var fs = require('fs');
+
+// Import request library to make requests over the network
+var request = require('request');
+
+// Import common helper
+var common = require('../dashboard/helper/common');
+
+// Set the node enviornment to test
+process.env.NODE_ENV = 'test';
+
+// Define user with parameterized constructor
+class User {
+    constructor(name, type, language, color, password, classroom) {
+        this.name = name;
+        this.type = type;
+        this.language = language;
+        this.color = color;
+        this.password = password;
+        if (classroom)
+            this.classroom = classroom;
+        this.status = 0;
+        this.comment = "";
+        this._id = "";
+    }
+}
+
+// Initialize the array that will contains all the users read from CSV
+var Users = [];
+
+// Trim and lowercase string
+function cleanString(string) {
+    if (!string) return;
+    string = string.trim();
+    string = string.toLowerCase();
+    return string;
+}
+
+// Validate color string
+function isValidColor(color) {
+    try {
+        color = JSON.parse(color);
+    } catch (e) {
+        return false;
+    }
+    if (!color.stroke || !color.fill) return false;
```

Comment:
Hmmm. Ideally stroke and fill should have value in the list of valid color set here;https://github.com/llaske/sugarizer-server/blob/dev/dashboard/helper/xocolors.js#L3

Reply:
Done:
https://github.com/llaske/sugarizer-server/blob/c020bd58e37b2b48d7071161efdf24d67fec0505/scripts/seed_users.js#L98-L103